src/caret_analyze/architecture/architecture.py

```diff
@@ -466,6 +466,41 @@ def rename_topic(self, src: str, dst: str) -> None:
         for c in self._communications:
             c.rename_topic(src, dst)
 
+    @staticmethod
+    def diff_node_names(
+            left_arch: Architecture,
+            right_arch: Architecture
+    ) -> Tuple[Tuple[str, ...], Tuple[str, ...]]:
+        return DiffArchitecture(left_arch, right_arch).diff_node_names()
+
+    @staticmethod
+    def diff_node_topics(
```

Comment:
This function compares all topic in Architecture.
Therefore, the name of this function should be,
```suggestion
    def diff_topic_names(
```

DiffArchitecture.diff_node_topics, too.